tests/ide-support/standalonebuilder/externalarchitectures.livecodescript

```diff
@@ -0,0 +1,71 @@
+﻿script "ExternalArchitectures"
+/*
+Copyright (C) 2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on TestSetup   
+   start using stack "revSBLibrary"
+end TestSetup
+
+on TestTeardown
+   
+end TestTeardown
+
+private function __ExtensionForPlatform pPlatform
+   switch pPlatform
+      case "Windows"
+         return ".dll"
+      case "Mac"
+         return ".bundle"
+      case "Linux"
+         return ".so"
+      case "iOS"
+      case "Android"
+         return ".lcext"
+      default
+         return empty
+   end switch
+end __ExtensionForPlatform
+
+private function __ArchitectureSynonyms pArchitecture
```

Comment:
am not overly happy with copying a private handler here but nor would I be happy with making it public so...